EncoreTickets.SDK.sln

```diff
@@ -7,6 +7,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EncoreTickets.SDK", "Encore
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EncoreTickets.SDK.Tests", "EncoreTickets.SDK.Tests\EncoreTickets.SDK.Tests.csproj", "{48098143-B237-4664-B18D-3E721C651871}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "CustomCodeAnalysis", "CustomCodeAnalysis", "{9256271B-CD0E-46C0-B0BD-FC5460F9F718}"
```

Comment:
What project is it?

Reply:
It's not a project, it's a solution folder

Reply:
for what? I do not see the use of this folder in PR

Reply:
This is to group the analyzer project together with the test project for the analyzer

Reply:
oh, sorry, now I see